src/components/Jobs/JobDescriptionTab.tsx

```diff
@@ -0,0 +1,87 @@
+import { Card, Divider, Group, Text } from '@mantine/core';
+
+import type { Job, WorkerMetadata } from '@/types/job';
+import { getUtcFromEpoch } from '@/utils/string';
+
+type props = {
+  jobData: { job: Job };
+};
+
+function JobDescriptionTab({ jobData }: props) {
+  const workers = jobData.job.workerMetadataList.map((workerMeta: WorkerMetadata) => (
+    <Card bg="gray.1" withBorder padding={0} radius="md" key={workerMeta.workerIndex}>
+      <Group position="apart" m={10}>
+        <Text>{workerMeta.jobId}</Text>
+        <Text>
+          CPU: <b> 2 </b> Ram: <b> 4000 MB </b> Disk: <b>100 MB</b> Network: <b>128 MBPS</b>
+        </Text>
+      </Group>
+      <Divider m={0} />
+      <Group position="left" m={10}>
+        <Text className={'font-bold'}>Worker Details -</Text>
+        <Text>
+          Worker index: {workerMeta.workerIndex} Worker Number: {workerMeta.workerNumber}
+        </Text>
+      </Group>
+      <Divider m={0} />
+      <div className={'grid grid-cols-[39%_2%_60%]'}>
+        <div className={'m-5  grid grid-cols-[25%_75%] justify-items-start gap-y-3'}>
+          <Text fw={700} c="dimmed">
+            Accepted:
+          </Text>
+          <Text>{getUtcFromEpoch(workerMeta.acceptedAt)}</Text>
```

Comment:
The same comment as earlier lets use date-fns format method to standardize on a specific date format that we show to users.